examples/a/disclosures.md

```diff
@@ -0,0 +1,128 @@
+__Disclosure for `jti`:__
+
+```
+WyJzSWhsbThYMXdhNGdXczd3eWg0V3VRIiwgImp0aSIsICJ1cm46Y3JlZGVudGlhbDo1
+YTRkNTQxMi0yN2UzLTQ1NDAtYTVlNS1mMWFhNGQ1NWIyMGMiXQ
+```
+
+Contents:
+
+```
+["sIhlm8X1wa4gWs7wyh4WuQ", "jti",
+"urn:credential:5a4d5412-27e3-4540-a5e5-f1aa4d55b20c"]
+```
+
+SHA-256 Hash: `MgwrNhUbZzmbEk_fvLJxVcObrgSjcdXO8pvwCnfE9Sg`
+
+__Disclosure for `binding`:__
+
+```
+WyJEUEpsSndjdXlhN2R0WWtlMlVJaDJRIiwgImJpbmRpbmciLCBbeyJ0eXBlIjogIkNs
+YWltc0JpbmRpbmRpbmcyMDIyIiwgImZhbWlsZU5hbWUiOiAiTXVzdGVybWFubiIsICJn
+aXZlbk5hbWUiOiAiRXJpa2EiLCAiZGF0ZU9mQmlydGgiOiAiMTk3My0wMS0wMSJ9XV0
+```
+
+Contents:
+
+```
+["DPJlJwcuya7dtYke2UIh2Q", "binding", [{"type":
+"ClaimsBindinding2022", "famileName": "Mustermann", "givenName":
+"Erika", "dateOfBirth": "1973-01-01"}]]
+```
+
+SHA-256 Hash: `iDpCqlGoMz0B_JOZozi-_G2XY9XelI1voUrNdj50HZY`
+
+__Disclosure for `achieved`:__
+
+```
+WyI2YnlLaDZTcHlzQkJFdFhaeWo3U0tRIiwgImFjaGlldmVkIiwgW3siaWQiOiAidXJu
+OmVwYXNzOmxlYXJuaW5nQWNoaWV2ZW1lbnQ6MSIsICJ0aXRsZSI6ICJNYXN0ZXIgb2Yg
+U2NpZW5jZSBpbiBDaXZpbCBFbmdpbmVlcmluZyIsICJ3YXNEZXJpdmVkRnJvbSI6IFt7
+ImlkIjogInVybjplcGFzczphc3Nlc3NtZW50OjEiLCAidGl0bGUiOiAiT3ZlcmFsbCBE
+aXBsb21hIEFzc2Vzc21lbnQiLCAiZ3JhZGUiOiAiZXhjZWxsZW50ICg1KSIsICJhc3Nl
+c3NlZEJ5IjogWyJkaWQ6ZWJzaTp6c1NnRFhlWVBoWjNBdUtoVEZuZURmMSJdLCAic3Bl
+Y2lmaWVkQnkiOiB7ImlkIjogInVybjplcGFzczphc3NzZXNzbWVudHNwZWM6MSIsICJ0
+aXRsZSI6ICJPdmVyYWxsIERpcGxvbWEgQXNzZXNzbWVudCIsICJncmFkaW5nU2NoZW1l
+IjogeyJpZCI6ICJ1cm46ZXBhc3M6c2NvcmluZ3NjaGVtZXNwZWM6MSIsICJ0aXRsZSI6
+ICJHZW5lcmFsIGdyYWRpbmcgc2NoZW1lIGluIENyb2F0aWEiLCAiZGVmaW5pdGlvbiI6
+ICJUaGUgQ3JvYXRpYW4gbmF0aW9uYWwgZ3JhZGluZyBzY2hlbWUgY29uc2lzdHMgb2Yg
+Zml2ZSBncmFkZXMgd2l0aCBudW1lcmljYWwgZXF1aXZhbGVudHM6IGl6dnJzdGFuIFx1
+MjAxMyA1IChvdXRzdGFuZGluZyk7IHZybG8gZG9iYXIgXHUyMDEzIDQgKHZlcnkgZ29v
+ZCk7IGRvYmFyIFx1MjAxMyAzIChnb29kKTsgZG92b2xqYW4gXHUyMDEzIDIgKHN1ZmZp
+Y2llbnQpOyBuZWRvdm9samFuIFx1MjAxMyAxIChpbnN1ZmZpY2llbnQgLSBmYWlsKS4g
+VGhlIG1pbmltdW0gcGFzc2luZyBncmFkZSBpcyBkb3ZvbGphbiBcdTIwMTMgMi4ifX19
+XX1dXQ
+```
+
+Contents:
+
+```
+["6byKh6SpysBBEtXZyj7SKQ", "achieved", [{"id":
+"urn:epass:learningAchievement:1", "title": "Master of Science in
+Civil Engineering", "wasDerivedFrom": [{"id":
+"urn:epass:assessment:1", "title": "Overall Diploma Assessment",
+"grade": "excellent (5)", "assessedBy":
+["did:ebsi:zsSgDXeYPhZ3AuKhTFneDf1"], "specifiedBy": {"id":
+"urn:epass:asssessmentspec:1", "title": "Overall Diploma
+Assessment", "gradingScheme": {"id":
+"urn:epass:scoringschemespec:1", "title": "General grading scheme in
+Croatia", "definition": "The Croatian national grading scheme
+consists of five grades with numerical equivalents: izvrstan \u2013
+5 (outstanding); vrlo dobar \u2013 4 (very good); dobar \u2013 3
+(good); dovoljan \u2013 2 (sufficient); nedovoljan \u2013 1
+(insufficient - fail). The minimum passing grade is dovoljan \u2013
+2."}}}]}]]
```

Comment:
I suggest an indented representation of this JSON for better readability

Reply:
This is auto generated by the script and I suppose there's no need to include this markdown snippet in the spec itself.